packages/context/src/lib/context-manager/shared/context.service.ts

```diff
@@ -588,7 +588,8 @@ export class ContextService {
           clusterParam: layerFound[`clusterParam`],
           visible: layer.visible,
           opacity: layer.opacity,
-          sourceOptions: layerFound.sourceOptions
+          sourceOptions: layerFound.sourceOptions,
+          workspace: layerFound.workspace
         };
```

Comment:
@pelord @aziz-access, On jase... pensez-vous qu'il y a une raison pour ne pas exporter tous les options du layer? Par exemple:
```suggestion
     const opts: AnyLayerOptions = {
          ...layerFound,
          title: layer.options.title,
          zIndex: layer.zIndex,
          style: layerStyle,
          visible: layer.visible,
          opacity: layer.opacity,
        };
```

Reply:
P-etre un lien avec l,API? aucune idée franchement, je ne me sers pas ce ce feature.

Reply:
@aziz-access Pense-tu que ce serait une bonne idée d'utiliser les modifs proposé dans mon commentaire. On délèguerait la responsabilité a l'API de bien gérer la donnée et nous on protègerais de future changements qu'on pourrait oublier.

Reply:
Oui bonne idée, on laisse la responsabilité à l'AP.